parsers/MN.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+
+from datetime import datetime
+from logging import Logger, getLogger
+from typing import Optional
+
+import arrow
+from bs4 import BeautifulSoup
+from requests import Session, get
+
+#   MAIN_WEBSITE = https://ndc.energy.mn/
+NDC_GENERATION = "https://disnews.energy.mn/test/convert.php"
+TZ = "Asia/Ulaanbaatar"  # UTC+8
+
+
+def fetch_production(
+    zone_key: str = "MN",
+    session: Optional[Session] = None,
+    target_datetime: Optional[datetime] = None,
+    logger: Logger = getLogger(__name__),
+):
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates.")
+    with get(NDC_GENERATION) as response:
+        NDC_SOUP = BeautifulSoup(response.content, "html.parser")
```

Comment:
Should this use session.get() instead?
```suggestion
    with session.get(NDC_GENERATION) as response:
        NDC_SOUP = BeautifulSoup(response.content, "html.parser")
```